conf/messages.en

```diff
@@ -600,6 +600,13 @@ partner.check.contact.email = Contact email
 partner.check.contact.phone = Phone number
 partner.check.contact.address = Contact address
 
+partner.name.title = What’s the name of the partner?
+partner.name.empty.error = Please enter your partner name
+partner.name.format.error = Please enter your partner name in correct format
+partner.name.title = What’s the name of the partner?
+partner.name.empty.error = Please enter your partner name
+partner.name.format.error = Please enter your partner name in correct format
+
```

Comment:
Again, I'd imagined we'd already have the necessary messages to capture a partnership name.

Reply:
Partner vs Partnership here.